terraform-runner/setup.sh

```diff
@@ -59,7 +59,8 @@ fi
 # Will not copy over "hidden" files (files that begin with '.').
 # Do not overwrite configmap
 mkdir -p $TFO_MAIN_MODULE
-false |  cp -iLr "$TFO_MAIN_MODULE_ADDONS"/* "$TFO_MAIN_MODULE" 2>/dev/null
+
+cp -Lr "$TFO_MAIN_MODULE_ADDONS"/* "$TFO_MAIN_MODULE" 2>/dev/null | true
```

Comment:
The combination of `false | cp -iLr ` makes sure to not overwrite any files in the target that already exist with the same name by filling in the prompt with the piped input `false`.  

I think making it failsafe is fine as I have no strong opinion against it. But I don't want to remove the "no overwrite" behavior... although now that I think about it, the "no overwrite" behavior was before the generational directory structure. Hm... Well I'll just suggest the simplest change which is to keep everything and add the pipe true.

```suggestion
false |  cp -iLr "$TFO_MAIN_MODULE_ADDONS"/* "$TFO_MAIN_MODULE" 2>/dev/null | true
```

Reply:
Ah, now I understand the hack. Okay, but, then we shouldn't pipe into "true".
"|| true" is the correct way.